manage.py

```
@@ -1,12 +1,13 @@
 #!/usr/bin/env python
 """Django's command-line utility for administrative tasks."""
+
 import os
 import sys
 
 
 def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)